apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -434,7 +439,7 @@ const PoolCandidatesTable = ({
           },
         }) => statusCell(poolCandidate.status, intl),
         meta: {
-          sortingLocked: true,
+          sortingLocked: !recordOfDecision,
           hideMobileHeader: true,
         },
       },
```

Comment:
I unlocked sorting for status and priority/category columns. I didn't unlock sorting for placement and final decision columns as I thought it would be confusing. Final decision and placement also access the status field, but then in the render step output something based off a function, therefore attempting to sort it wouldn't look sensible. 

Example

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/86f773b7-fc69-4e9d-bf1b-17c191092c67)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/c2a29228-683b-45f9-ad16-daca14526228)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/a05c8278-b213-4a69-b786-e22019c94a4a)

Reply:
I'm wondering it is better to use [`enableSorting`](https://tanstack.com/table/v8/docs/api/features/sorting#enablesorting) instead. I believe the lock is a visual indication that the table is sorted by that column and cannot be changed which is technically not true.

Reply:
Sure, changed the other two columns to `enableSorting = false` for improved clarity in https://github.com/GCTC-NTGC/gc-digital-talent/pull/9318/commits/4541d41fa64882ae8b5c8fde164dc256c07d3d92